graphrag_sdk/test_metrics.py

```
@@ -0,0 +1,545 @@
+from typing import Optional
+from deepeval.metrics import BaseMetric, AnswerRelevancyMetric, FaithfulnessMetric, ContextualRelevancyMetric
```

Ruff (0.8.2)

`deepeval.metrics.ContextualRelevancyMetric` imported but unused

Remove unused import: `deepeval.metrics.ContextualRelevancyMetric`

(F401)